types/marker.go

```diff
@@ -15,3 +17,53 @@ func NewMarkerParams(params markertypes.Params, height int64) *MarkerParams {
 		Height: height,
 	}
 }
+
+// MarkerAccount represents the x/marker marker account
+type MarkerAccount struct {
+	Address                string
+	AccessControl          []markertypes.AccessGrant
+	AllowGovernanceControl bool
+	Denom                  string
+	MarkerType             markertypes.MarkerType
+	Status                 markertypes.MarkerStatus
+	Supply                 []MarkerSupply
```

Comment:
would it be good if we set the `Supply` 's type as `sdk.Coins` instead of `[]MarkerSupply` for better readibility and reduce additional type struct?